lte/gateway/c/oai/tasks/s1ap/s1ap_mme.c

```diff
@@ -292,9 +285,6 @@ static int handle_message(zloop_t* loop, zsock_t* reader, void* arg) {
        * Need to handle it when we support other timers in S1AP
```

Comment:
Remove lines 291-294, as the comment is for function in lines 295-297

Reply:
Resolved